v_4_9_0/cloudconfig.go

```diff
@@ -1,4 +1,4 @@
-package v_4_8_0
+package v_4_9_0
```

Comment:
This should be done in a separate PR. So the adding of `4_9_0` is separate from the actual change.

Reply:
So three PRs: copy previous version, rename package, populate changes?

Reply:
Yes - so it's a bit more easy to see which the actual changes are. Not much but a little :P

Reply:
Heh, okay. I did do it in separate commits, but I will revert and split into another PR.